services/ows_refactored/radar_backscatter/ows_sentinel1_cfg.py

```diff
@@ -0,0 +1,91 @@
+    },
+}
+
+
+style_s1_vh_over_vv= {
+    "name": "vv_vh_vv_over_vv",
+    "title": "VV, VH and VH/VV",
+    "abstract": "False colour representation of VV, VH and VH/VV for R, G and B respectively",
+    "additional_bands": [],
+    "components": {
+        "red": {"vv":1.0,"scale_range":[0.0,0.28]},
+        "green": {"vh":1.0,"scale_range":[0.0,0.06]},
+        "blue": {
+            "function": "datacube_ows.band_utils.band_quotient",
+            "mapped_bands": True,
+            "kwargs": {"band1": "vh", "band2": "vv", "scale_from":[0.0,0.49]},
+        },
+    },
+}
+
+layer = {
+    "title": "Normalized Radar Backscatter Sentinel-1",
+    "name": "s1_rtc",
+    "abstract": """
+
+Synthetic Aperture Radar (SAR) data have been shown to provide different and complementary information to the more common optical remote sensing data. Radar backscatter response is a function of topography, land cover structure, orientation, and moisture characteristics—including vegetation biomass—and the radar signal can penetrate clouds, providing information about the earth’s surface where optical sensors cannot. Digital Earth Africa provides access to Normalized Radar Backscatter data, for which Radiometric Terrain Correction (RTC) has been applied so data acquired with different imaging geometries over the same region can be compared. 
+
+The twin Sentinel-1 satellites, launched in 2014 and 2016, are operated by the European Space Agency (ESA) as part of European Commission's Copernicus Programme. They currently collects C-band SAR data every 12 days over Africa. 
+
+This product contains radar measurement in C-band and in VV and VH polarizations. It has a spatial resolution of approximate 20 m, a temporal coverage of 2017 to current and is updated as new images are acquired. Data is provided as Gamma0 in linear power, which can be converted to backscatter in decibel unit using 10*log10(DN).
+
+This dataset is processed by Sinergise Sentinel Hub using Sentinel-1 GRD product provided by ESA as input. RTC has been calcuated using 30 m Copernicus Digital Elevation Model.
+
+More techincal information about the Sentinel-1 Normalized Backscatter can be found in the User Guide (https://docs.digitalearthafrica.org/en/latest/data_specs/Sentinel-1_specs.html)
+
+This product is accessible through OGC Web Service (https://ows.digitalearth.africa/), for analysis in DE Africa Sandbox JupyterLab (https://github.com/digitalearthafrica/deafrica-sandbox-notebooks/wiki) and for direct download from AWS S3 (https://data.digitalearth.africa/).
+""",
+    "product_name": "s1_rtc",
+    "time_resolution": "year",
+    "bands": bands_s1,
+    "resource_limits": reslim_alos_palsar, #the argument need to be checked as it was already set to be SRTM 
+    "image_processing": {
+        "extent_mask_func": "datacube_ows.ogc_utils.mask_by_val",
+        "always_fetch_bands": [],
+        "manual_merge": False,
+    },
+    "flags": {
```

Comment:
You are defining a flag band using the old deprecated syntax, but you aren't actually using it in any of the styles.

Do you need help with applying masks?

Flags should be in a list:
```
    "flags": [
          { ... what you have here ... },
    ],
```

Reply:
I'm not sure whether I should include them or not. We have a mask band so it would be an advantage to include that I guess. I believe that I need help with how to apply masks.